packages/blade/src/utils/makeMotionTime/makeMotionTime.all.ts

```diff
@@ -0,0 +1,5 @@
+import type { MakeMotionTime } from './types';
```

Comment:
Unsure about why we need `makeMotionTime.all`. Tried things without this and cast* functions seem to be returning things appropriately. Let's catch up for this?

Reply:
It's not actually returning the correct types. 

Let's remove the .all file and then cast the type to native. 

<img width="778" alt="image" src="https://user-images.githubusercontent.com/35374649/190356802-85539493-a475-4819-bfe9-9b38906b3871.png">

You'll see that castNativeType is returning `never`, It's returning never since without the all type the module resolution will only resolve the `.web` type and the Platform.CastNative's `Extract<>` will fail. 

And this will happen vice versa, when your module resolution is set to `.native` if you try to do `castWebType` the type will return `never`

----

This is particularly bad also because `never` can be assignable to all the types and will bite us later, since it won't throw error:

```ts
const t: number = 'this is not a number' as never;
```